packages/core/src/RepoSync.test.ts

```diff
@@ -112,7 +109,7 @@ describe(`RepoSync`, () => {
       await makeConnection('L', localRepo, channel)
 
       localRepo.set('xyz', A.from({ boo: 999 }, 'L'))
-      await docChanged(remoteRepo)
+      await _yield()
```

Comment:
a 100-ms yield is more robust to changes in implementation details than listening for change events.